main/options.c

```diff
@@ -57,7 +57,7 @@
 #define EXTENSION_SEPARATOR '.'
 #define PATTERN_START '('
 #define PATTERN_STOP  ')'
-#define IGNORE_SEPARATORS   ", \t\n"
```

Comment:
All macros can contain spaces and , characters.
Maybe we should look into a way of keeping -I backward-compatible and -D supporting full macros.
Though this can be done in steps. In the long term I would also like to parse real macros, not the "fake" ones with $ characters.